lib/src/widgets/SnackBars/noNetworkConnectionSnackBar.dart

```diff
@@ -1,14 +1,14 @@
-import 'package:exerlog/UI/global.dart';
+import 'package:exerlog/src/core/theme/theme_color.dart';
 import 'package:flutter/material.dart';
 
 SnackBar noNetworkConnectionSnackBar() => SnackBar(
-      backgroundColor: redBackgroundColor,
+      backgroundColor: ThemeColor.redBackgroundColor,
```

Comment:
Use the colors from the theme provided by the ThemeProvider widget

Reply:
@jorgelrj yes I konw, but i thought that color name was important :)